Sage/carbon
# fix(select): resolved error when only one option child is set - FE-2547

Fixes FE-2547
Fixes GitHub issue #2567

### Proposed behaviour
Select can accept one or more `<Option />` as children.

### Current behaviour
Select breaks when one `<Option />` is given as child.

### Checklist


<dev>- [ ] Cypress automation tests
<dev>- [ ] Storybook added or updated
<dev>- [ ] Theme support
<dev>- [ ] Typescript `d.ts` file added or updated

### Testing instructions
To experience breaking behaviour, on master branch modify the `defaultStory` story in `select.stories.js`.

```diff
const defaultComponent = () => {
   return (
     <State store={ singleSelectStore }>
       <Select ariaLabel='singleSelect' { ...commonKnobs(singleSelectStore) }>
-        { selectOptions }
+        <Option
+          key="a"
+          text="a"
+          value="1"
+        />
       </Select>
     </State>
   );
);
```

browser to:
http://localhost:9001/?path=/story/experimental-select--default
click on select, it should raise the error in the browser console.

Checkout this branch and repeat the steps. There should be no errors.
